src/commands/config.rs

```diff
@@ -142,6 +144,11 @@ pub fn load_config() -> Config {
     Config::from_iter(strip_server(std::env::args()).iter())
 }
 
+fn parse_socket_addr(s: &str) -> std::io::Result<SocketAddr> {
+    let mut addrs = s.to_socket_addrs()?;
+    Ok(addrs.next().unwrap())
```

Comment:
I think this might panic if the host did not have networking enabled (or the address couldn't be resolved for some other reason)?

https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html#tymethod.to_socket_addrs

> The returned iterator may not actually yield any values depending on the outcome of any resolution performed.

I suggest at least giving the user a hint when this happens with something like;

```suggestion
    Ok(addrs.next().expect("Could not resolve string to any socket address. Is networking enabled?"))
```

Reply:
makes sense